templates/company/about.html

```diff
@@ -0,0 +1,464 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/19TIwT-GU5wADQmohicj14MnpHEqNqp0422WLunFsbJU
+{% endblock meta_copydoc %}
+
+{% block title %}Canonical - company information{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}company/about{% endblock %}
+
+{% block meta_description %}
+  About Canonical |The publisher of Ubuntu and provider of open source security, support and services
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <div class="u-fixed-width">
+    <nav class="p-breadcrumbs" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+        <li class="p-breadcrumbs__item ">
+          <a href="/company">Company</a>
+        </li>
+        <li class="p-breadcrumbs__item ">About Canonical</li>
+      </ol>
+    </nav>
+    <hr class="p-rule" />
+  </div>
+
+  <section class="p-section--hero">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <h1 class="u-no-margin--bottom">About Canonical</h1>
+        <h2>
+          Open source offers the world’s innovation in code. We deliver it faster, more securely and more cost-effectively than any other company.
+        </h2>
+      </div>
+      <div class="p-cta-block">
+        <a href="/contact-us" class="p-button--positive">Contact our team</a>
```

Comment:
This isn't specified on the copy doc, was the link confirmed on MM and if so could you update the doc to reflect?

Reply:
@juanruitina can you pls confirm this link?

Reply:
Makes sense, but I'm asking Gloria to confirm in the spec anyway.